test/awsRunInitBootstrap.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
```

Comment:
I'd be hopeless at finding a bug in this test code, could we use python or typescript (via node)?

Reply:
Yeah looking back now it may be beneficial to write this in python, I will create a task for this

Reply:
https://opensearch.atlassian.net/browse/MIGRATIONS-2231